python/tidy/servo_tidy/tidy.py

```diff
@@ -360,6 +351,43 @@ def find_reverse_dependencies(dependency, version, content):
 """.format(**substitutions).strip()
                 yield (1, message)
 
+        # Packages with same version from different sources
+        if (len(set(data["version"])) < len(data["version"])
```

Comment:
Maybe we should have names for version and source, so that we don't have to address them as `data["version"]` and `data["source"]` everywhere.